src/utils/requester-configurations/put.ts

```diff
@@ -27,6 +27,7 @@ export default {
   headers: {},
   data: {},
   json: true,
+  timeout: 10000,
```

Comment:
How about extracting this timeout value into some constant so we could re-use it across the app?

Reply:
Makes sense, I would also like to add different value if test is running and pass it to unit tests 👍